labrad/util/mux.py

```diff
@@ -0,0 +1,66 @@
+from twisted.internet import defer
+
+from labrad import util
+
+
+def select(**kwargs):
```

Comment:
Changed to take a dict of options instead of using keyword args. Also added a helper function to produce wakeup calls, instead of having separate support for timeouts, so that the timeout key/Deferred is treated exactly like any other selection option.